tests/phpunit/tests/media.php

```diff
@@ -2467,6 +2467,9 @@ public function test_wp_calculate_image_srcset_animated_gifs() {
 	 * @requires function imagejpeg
 	 */
 	public function test_wp_filter_content_tags_schemes() {
+		// Disable lazy loading attribute.
+		add_filter( 'wp_img_tag_add_loading_attr', '__return_false' );
```

Comment:
Worth clarifying why:
```suggestion
		// Disable lazy loading attribute to not add the 'auto' keyword to the `sizes` attribute.
		add_filter( 'wp_img_tag_add_loading_attr', '__return_false' );
```